HoloRepositoryDemoApplication/Assets/HoloRpository/Scripts/StorageConnectionServer.cs

```diff
@@ -0,0 +1,256 @@
+﻿using System;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.Networking;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization;
+using System.Collections.Generic;
+using System.Reflection;
+using SimpleJSON;
+
+namespace HoloRepository
+{
+    public class StorageConnectionServer : MonoBehaviour
+    {
+        #region Properties
+        private static string BaseUri = "http://localhost:3001/api/v1";
```

Comment:
As in the `ModelLoader`, it may be useful to have this configurable from the inspector. Also port and maybe even API version could be handled separately.

Reply:
For now, This package is only used as script API, so these scripts won't be add to the unity hierarchy

Reply:
Hm ok... but maybe domain, port and api version could be three variables, and the BaseUri could interpolate one string (see [here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated)).